roomserver/api/query.go

```diff
@@ -251,8 +257,8 @@ type QueryRoomVersionCapabilitiesRequest struct{}
 
 // QueryRoomVersionCapabilitiesResponse is a response to QueryServersInRoomAtEventResponse
 type QueryRoomVersionCapabilitiesResponse struct {
```

Comment:
Docs make no sense. It's a response to a response..? Why do we have this response struct at all? I thought we were planning on having versions threaded to other responses like we have done in this PR?

Reply:
Typo in comments, will fix. Also this is a different request - the one that allows asking the roomserver for *supported* room versions for `capabilities` requests.

Reply:
Right, gotcha!